facdb/sql/_deduplication.sql

```diff
@@ -19,3 +19,13 @@ AND bin in (
 	SELECT distinct bin FROM facdb
 	WHERE datasource = 'dfta_contracts'
 );
+
+-- Remove out of NYC zipcodes
+DELETE FROM facdb WHERE zipcode NOT IN (SELECT zipcode FROM doitt_zipcodeboundaries);
```

Comment:
This won't work since doitt_zipcodeboundaries doesn't contain all valid zipcodes in NYC (i.e. 10150).  I think the step below captures what needs to be deleted because it's outside nyc

Reply:
those are po box zipcodes, do we have to keep those records? chances are they don't have geometries anyways